models/cbcsrf.cfc

```diff
@@ -44,10 +44,15 @@ component accessors="true" singleton {
 	 *
 	 * @return The csrf token
 	 */
-	public string function generate( string key = "default", boolean forceNew = false ){
+	public string function generate( string key, boolean forceNew = false ){
 		// Get our session csrf data
 		var csrfData = cacheStorage.get( getTokenStorageKey(), {} );
 
+		// Mixins pass an empty key argument so "default" isn't set and verification fails when using the examples given in readme.md
+		if (!key.len()){
```

Comment:
Can you add an is null check as well please

```
if( !isNull( arguments.key ) && !key.len() ){
  arguments.key = "default"
}
```

Reply:
Added.  I don't know how this "change to a PR" thing works though - I hope I did it right.